code/modules/mob/living/carbon/slime/slime.dm

```diff
@@ -204,6 +204,12 @@
 	updatehealth()
 	return
 
+/mob/living/carbon/slime/MouseDrop(var/atom/movable/A as mob|obj)
+	if(isliving(A) && A != usr)
+		var/mob/living/Food = A
+		if(Food.Adjacent(usr) && !stat && Food.stat != DEAD) //messy
```

Comment:
oh, i looked at it again and noticed the living/canusetopic. alright, i'll pop it in and see if it works.